RevenueCatUI/CustomerCenter/Views/ManageSubscriptionsView.swift

```diff
@@ -382,7 +387,6 @@ struct SubscriptionDetailsView_Previews: PreviewProvider {
         )
         .previewDisplayName("Subscription Details - Monthly")
         .padding()
-
```

Comment:
According to our swift style guide we keep a line at the top and bottom of types